ngraph/python/tests/test_onnx/utils/__init__.py

```diff
@@ -28,8 +28,43 @@
 from tests.test_onnx.utils.onnx_helpers import import_onnx_model
 
 
-def xfail_test(*backends, reason="Mark the test as expected to fail"):
-    return pytest.mark.xfail(condition=tests.BACKEND_NAME in backends, reason=reason, strict=True)
+def xfail_test(reason="Mark the test as expected to fail", strict=True):
+    return pytest.mark.xfail(reason=reason, strict=strict)
+
+
+segfault = pytest.mark.skip(reason="Segmentation fault error")
```

Comment:
Let's rename some od these decorators, to make it more obvious what they mean.
```suggestion
skip_segfault = pytest.mark.skip(reason="Segmentation fault error")
```

Reply:
done